Samsung/a54x/res/values/config.xml

```diff
@@ -0,0 +1,2260 @@
+        <item>956.863</item>
+        <item>958.824</item>
+        <item>960.784</item>
+        <item>962.745</item>
+        <item>964.706</item>
+        <item>966.667</item>
+        <item>968.627</item>
+        <item>970.588</item>
+        <item>972.549</item>
+        <item>974.51</item>
+        <item>976.471</item>
+        <item>978.431</item>
+        <item>980.392</item>
+        <item>982.353</item>
+        <item>984.314</item>
+        <item>986.275</item>
+        <item>988.235</item>
+        <item>990.196</item>
+        <item>992.157</item>
+        <item>994.118</item>
+        <item>996.078</item>
+        <item>998.039</item>
+        <item>1000</item>
+    </array>
+    <integer-array name="config_screenBrightnessRangeForClearView">
+        <item>255</item>
+        <item>486</item>
+    </integer-array>
+<integer-array name="config_availableColorModes">
+        <item>0</item>
+        <item>1</item>
+        <item>2</item>
+        <item>3</item>
+    </integer-array>
+    
+    <bool name="config_dynamic_automatic_brightness_available">true</bool>
+    <bool name="config_dozeAlwaysOnDisplayAvailable">true</bool>
+    <bool name="config_showNavigationBar">true</bool>
+    <bool name="config_supportAudioSourceUnprocessed">false</bool>
+    <bool name="config_systemCaptionsServiceCallsEnabled">true</bool>
+      <bool name="config_carrier_volte_available">true</bool>
+    <bool name="config_device_volte_available">true</bool>
+    <bool name="config_enableExynosDisplaySolutionService">true</bool>
+    <bool name="config_enableWifiDisplay">true</bool>
+    <bool name="config_flipToScreenOffEnabled">false</bool>
+    <bool name="config_hotswapCapable">true</bool>
+    <bool name="config_setColorTransformAccelerated">true</bool>
+    <bool name="config_useDevInputEventForAudioJack">true</bool>
+    
+     <item type="dimen" name="config_screenBrightnessDimFloat">0.043137256</item>
+    <item type="dimen" name="config_screenBrightnessSettingDefaultFloat">0.5019608</item>
+    <item type="dimen" name="config_screenBrightnessSettingMaximumFloat">1.0</item>
+    <item type="dimen" name="config_screenBrightnessSettingMinimumFloat">0.0</item>
+    <dimen name="cover_clear_view_bottom">242.0dip</dimen>
+    <dimen name="cover_clear_view_left">204.0dip</dimen>
+    <dimen name="cover_clear_view_right">360.0dip</dimen>
+    <dimen name="cover_clear_view_rounded_corner">33.0dip</dimen>
+    <dimen name="cover_clear_view_top">1.0dip</dimen>
+    <dimen name="status_bar_camera_side_padding">30.0px</dimen>
+    <dimen name="status_bar_camera_top_margin">22.0px</dimen>
+    <dimen name="status_bar_height_portrait">80.0px</dimen>
+    <dimen name="status_bar_indicator_corner_padding">60.0px</dimen>
+      <string name="config_mainBuiltInDisplayCutout">M 0,0 M 0,7.822222222222222‬‬ a 10.31111111111111‬,10.31111111111111‬ 0 1,0 0,20.62222222222222‬ a 10.31111111111111‬,10.31111111111111‬ 0 1,0 0,-20.62222222222222‬ Z @dp</string>
```

Comment:
Remove hidden unicode chars

Reply:
I can't seem to remove them no matter what I try.

Reply:
UPDATE: I managed to remove the hidden Unicode here.